Plugins/Cirrious/File/Cirrious.MvvmCross.Plugins.File.Wpf/Plugin.cs

```diff
@@ -5,18 +5,40 @@
 // 
 // Project Lead - Stuart Lodge, @slodge, me@slodge.com
 
+using System;
+using System.IO;
 using Cirrious.CrossCore;
+using Cirrious.CrossCore.Platform;
 using Cirrious.CrossCore.Plugins;
 
 namespace Cirrious.MvvmCross.Plugins.File.Wpf
 {
     public class Plugin
-        : IMvxPlugin          
+        : IMvxConfigurablePlugin
     {
+        private string _rootFolder;
+
         public void Load()
         {
-            Mvx.RegisterType<IMvxFileStore, MvxWpfFileStore>();
-            Mvx.RegisterType<IMvxFileStoreAsync, MvxWpfFileStore>();
+            var rootFolder = _rootFolder ?? Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
```

Comment:
Something needs to set _rootFolder at some point - it's not really a "bug", but I don't like having `_rootFolder` left dangling as `null`